tests/playwright/Report.md

```diff
@@ -0,0 +1,213 @@
+# Evaluation of Microsoft Playwright
+
+## Table of Contents
+
+[Introduction](#Introduction)
+
+[Project Background](#project-background)
+
+[Playwright Vs WebdriverIO](#playwright-vs-webdriverIO)
+
+* [Sample Playwright Page screenshot code](#sample-playwright-page-screenshot-code)
```

Comment:
If you have sample playwright code, it would make sense to have the same sample code in webdriverio.

Reply:
Noted